### ISSUE TITLE
Duckduckgo and ExaSearch do not work 

### ISSUE THREAD

**Issue by IvanLee113** [crewAIInc/crewAI#384]
Hello all,

I have been juggling between experimenting Exa search and Duckduckgo in prep for meeting example, but none of them work. 

For duckduckgo, I am experiencing the same issue in the thread below
https://github.com/joaomdmoura/crewAI/issues/332

For Exa Search, it'll do a bunch of loop like below, before eventually producing empty results. Yes, I have set the api key in .env already.

```
Thought: Given the meeting context of planning a teambuilding event in Berlin that involves bars and nightclubs, we need to develop strategic talking points, questions, and discussion angles that will help us achieve the meeting objective.


Action: Search for information on successful teambuilding events in Berlin that involve bars and nightclubs using the search tool.
Action Input: {'query': 'teambuilding events in Berlin', 'Tool': 'search'}

Action 'Search for information on successful teambuilding events in Berlin that involve bars and nightclubs using the search tool.' don't exist, these are the only available Actions: search: search(query) - Search for a webpage based on the query.
find_similar: find_similar(url: str) - Search for webpages similar to a given URL.
                The url passed in should be a URL returned from `search`.
get_contents: get_contents(ids) - Get the contents of a webpage.
                The ids must be passed in as a list, a list of ids returned from `search`.
Delegate work to co-worker: Delegate work to co-worker(coworker: str, task: str, context: str) - Delegate a specific task to one of the following co-workers: ['Research Specialist', 'Industry Analyst', 'Briefing Coordinator']
The input to this tool should be the coworker, the task you want them to do, and ALL necessary context to exectue the task, they know nothing about the task, so share absolute everything you know, don't reference things but instead explain them.
Ask question to co-worker: Ask question to co-worker(coworker: str, question: str, context: str) - Ask a specific question to one of the following co-workers: ['Research Specialist', 'Industry Analyst', 'Briefing Coordinator']
The input to this tool should be the coworker, the question you have for them, and ALL necessary context to ask the question properly, they know nothing about the question, so share absolute everything you know, don't reference things but instead explain them.
```

```
Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.

Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.

Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event"Thought: As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting.
Action: Search
Action Input: query = "Berlin team building event" As the Briefing Coordinator, I should compile all gathered information into a concise, informative briefing document for the meeting. To do this, I will search for relevant information on team building events in Berlin.

Action: Search
Action Input: query = "Berlin team building event"
```

I am running on local llama2. Please help :(

**Comment by menguzat** [comment]
I have a similar issue with llama3

Thought: I need to start by searching for general information about the wonders of the world. 

Action: duckduckgo_search

Action Input: {"query": "wonders of the world", "context": "general informationI apologize for the mistake. Let me try again!

Thought:
Since I need to start by searching for general information about the wonders of the world, I'll use the duckduckgo_search tool.

Action: duckduckgo_search
Action Input: {"query": "wonders of the world", "context": "general informationI apologize for the mistake earlier! Let me try again with the correct input.

Thought:

this goes on forever.
